cli/dcoscli/data/help/package.txt

```diff
@@ -3,73 +3,82 @@ Install and manage DCOS packages
 Usage:
     dcos package --config-schema
     dcos package --info
+    dcos package create [--output-directory=<output-directory>]
```

Comment:
Potential alternate name: `bundle` instead of create.

This to me is more clear that it's bundling a directory (manifest and artifacts) into a dcos package.  When I saw `create` is imagined this was more akin to "setup the directory structure for me".